concepticondata/conceptlists/Steinthal-1875-1565.tsv

```diff
@@ -0,0 +1,1566 @@
+Steinthal-1875-1565-118	118	Flamme	Flamme	flame	889	FLAME
+Steinthal-1875-1565-119	119	Rauch steigt auf.	Rauch steigt auf.	Smoke rises.		
+Steinthal-1875-1565-120	120	Feuerkohle	Feuerkohle	charcoal	1188	CHARCOAL
+Steinthal-1875-1565-121	121	Asche	Asche	ash	646	ASH
+Steinthal-1875-1565-122	122	Metall (die gewöhnlichen Metalle)	Metall	metal	2244	METAL
+Steinthal-1875-1565-123	123	Sand	Sand	sand	671	SAND
+Steinthal-1875-1565-124	124	Lehm	Lehm	silt	1399	CLAY
+Steinthal-1875-1565-125	125	Stein (Edelsteine, Kiesel, Feuerstein)	Stein	stone	857	STONE
+Steinthal-1875-1565-126	126	Stein (Edelsteine, Kiesel, Feuerstein)	Edelsteine	gemstone	1029	JEWEL
+Steinthal-1875-1565-127	127	Stein (Edelsteine, Kiesel, Feuerstein)	Kiesel	pebble	961	PEBBLES
+Steinthal-1875-1565-128	128	Stein (Edelsteine, Kiesel, Feuerstein)	Feuerstein	flintstone		
+Steinthal-1875-1565-129	129	Staub	Staub	dust	2	DUST
+Steinthal-1875-1565-130	130	Schlamm	Schlamm	mud	640	MUD
+Steinthal-1875-1565-131	131	Salz	Salz	salt	1274	SALT
+Steinthal-1875-1565-132	132	Seife	Seife	soap	788	SOAP
+Steinthal-1875-1565-133	133	Baum fällen	Baum fällen	to fell a tree		
+Steinthal-1875-1565-134	134	Strauch	Strauch	shrub	751	SHRUB
+Steinthal-1875-1565-135	135	Rohr abschneiden	Rohr abschneiden	to cut reed		
+Steinthal-1875-1565-136	136	Gras	Gras	grass	606	GRASS
+Steinthal-1875-1565-137	137	Der Baum treibt Blüthen.	Der Baum treibt Blüthen.	The tree bears blossoms.		
+Steinthal-1875-1565-138	138	Das Kraut schiesst auf. 	Das Kraut schiesst auf. 	The plants shoot up.		
+Steinthal-1875-1565-139	139	Getreide (dessen übliche Arten)	Getreide	grain	605	GRAIN
+Steinthal-1875-1565-140	140	Arten von Hülsenfrüchten, von Gemüsen	Hülsenfrüchte	legumes		
+Steinthal-1875-1565-141	141	Arten von Hülsenfrüchten, von Gemüsen	Gemüse	vegetables	924	VEGETABLES
+Steinthal-1875-1565-142	142	Zweig	Zweig	twig	3222	TWIG
+Steinthal-1875-1565-143	143	Blatt	Blatt	leaf	628	LEAF
+Steinthal-1875-1565-144	144	Laub	Laub	leaves		
+Steinthal-1875-1565-145	145	Rinde	Rinde	bark	1204	BARK
+Steinthal-1875-1565-146	146	Dorn	Dorn	thorn	124	THORN
+Steinthal-1875-1565-147	147	Die Frucht reift, ist noch nicht reif, faul.	Die Frucht reift.	The fruit ripens.	179	RIPEN
+Steinthal-1875-1565-148	148	Die Frucht reift, ist noch nicht reif, faul.	Die Frucht ist noch nicht reif.	The fruit is not ripe yet.	1958	UNRIPE
+Steinthal-1875-1565-149	149	Die Frucht reift, ist noch nicht reif, faul.	Die Frucht fault.	The fruit rots.		
+Steinthal-1875-1565-150	150	Same	Same	seed	714	SEED
+Steinthal-1875-1565-151	151	Wurzel	Wurzel	root	670	ROOT
+Steinthal-1875-1565-152	152	Kartoffel	Kartoffel	potato	593	POTATO
+Steinthal-1875-1565-153	153	Holz spalten	Holz spalten	to split wood		
+Steinthal-1875-1565-154	154	Stroh	Stroh	straw	2299	STRAW
+Steinthal-1875-1565-155	155	Heu	Heu	hay	342	HAY
+Steinthal-1875-1565-156	156	Palme	Palme	palm tree	1181	PALM TREE
+Steinthal-1875-1565-157	157	Dattel	Dattel	date		
+Steinthal-1875-1565-158	158	Kaffee	Kaffee	coffee		
```

Comment:
Not 1299 because this list item here refers to the plant, correct?

Reply:
Yes. Or I assumed so, because the surrounding concepts are referring to plants and fruits as well.